### ISSUE TITLE
Add ability to load default preferences

### ISSUE THREAD

**Issue by rob-luke** [cbrnr/mnelab#175]
I would like to be able to feed a default set of parameters to mnelab when it opens. The values I use for analysing and viewing EEG vs NIRS files are completely different. Would it be possible to have some sort of config file that specifies defaults for viewing and analysing files? The simplest example is the time window that I want to view my data in, NIRS I like to see around 10 min at a time, whereas I like to view less than a minute of EEG in the raw viewer.

This is a feature suggestion, it may be simply to complex to implement. I can't implement this, but thought it would be worth suggesting incase it can be slowly worked in with other PRs.

**Comment by cbrnr** [comment]
I think that's a very good idea. Maybe the simplest solution would be to support profiles, which can either be loaded on startup (e.g. via command line parameters), or activated/loaded from MNELAB at anytime. However, I think it is important to be very clear what goes into a profile - what would you suggest?

Since this solution is all or nothing, what about users who would want to load only some specific options but not a whole profile? Maybe in this case MNELAB could present a dialog where all supported options can be checked/unchecked.

**Comment by rob-luke** [comment]
> I think it is important to be very clear what goes into a profile - what would you suggest?

The first two things that come to mind are the span of time shown in the waveform viewer and default filter parameters.

> Since this solution is all or nothing, what about users who would want to load only some specific options but not a whole profile?

Could we maintain the current default parameters, and the profile overwrites defaults if present? Or would this get messy?

> Maybe in this case MNELAB could present a dialog where all supported options can be checked/unchecked.

Thats nice, then an export button to save the profile to disk for reuse?

**Comment by cbrnr** [comment]
Yes, something like that. I think it should be possible to use a profile/config both from the command line and from the GUI.
